src/hooks/useImage.ts

```diff
@@ -36,8 +36,8 @@ const useImage = (
         mutation.mutate(formData, {
           onSuccess: (data) => {
```

Comment:
p5;
구조분해할당 사용해도 좋을 것 같네요!
```suggestion
          onSuccess: ({photoId, photoUrl}) => {
```

Reply:
적용할게요~